### DISCUSSION TITLE
命令行生效设置

### DISCUSSION THREAD

**Discussion by bczhc** [fcitx/fcitx5#746] [Q&A] [answered]
在黑暗模式下想要也改变一下fcitx5的主题，修改了`.config/fcitx5/conf/classicui.conf`之后，貌似不会自动更新设置。除了fcitx5 -r之外还能快速应用设置吗？因为fcitx5等于是重启fcitx5，没这个必要。

**Comment by wengxt** [comment, marked as answer]
`qdbus org.fcitx.Fcitx5 /controller org.fcitx.Fcitx.Controller1.ReloadAddonConfig classicui`

`dbus-send --print-reply=literal --dest=org.fcitx.Fcitx5 /controller org.fcitx.Fcitx.Controller1.ReloadAddonConfig string:classicui`

`busctl --user call org.fcitx.Fcitx5 /controller org.fcitx.Fcitx.Controller1 ReloadAddonConfig s classicui`

**Reply by bczhc** [reply]
三种方法，厉害！
